src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -343,7 +343,7 @@ public void OnFinishEditing()
 
         // Update membrane
         editedSpecies.MembraneType = Membrane;
-        editedSpecies.Colour = Colour;
+        editedSpecies.Colour = gui.GetMembraneColor();
```

Comment:
Doesn't this mean that the Colour property / field from MicrobeEditor class should be removed? As it isn't used for anything now.